README.md

```diff
@@ -224,6 +249,9 @@ TBC
 * [VBA E-Book for beginners](https://goalkicker.com/VBABook/)
 * [Excel VBA E-Book for beginners](https://goalkicker.com/ExcelVBABook/)
 * [Rubberduck Blog](https://rubberduckvba.wordpress.com/) - Intermediate-Advanced topics.
+* [📧][a_ol] [Slipstick](https://www.slipstick.com/) - Website of Diane Poremsky (MVP) with focus on Outlook and VBA. 
```

Comment:
In 2 minds about including this in both Websites and Forums... Seems a little unfair.  Then again, it is difficult to find the forum on Slipstick site...